web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1031,6 +1032,9 @@ fixmystreet.update_pin = function(lonlat, savePushState) {
             category_select.change();
         }
         fixmystreet.run(fixmystreet.set_up.category_groups);
+        if (category_group !== category_select.val()) {
```

Comment:
This doesn't feel right to me, e.g. it is called if a category+subcategory were already selected, changing the group to what it is already set to. Should it not be `if (category_group !== '-- Pick a category --' && category_select.val() == '-- Pick a category --') {`

Reply:
Ah right, I see your point.